### PR TITLE
fix form

### PR DESCRIPTION
**Pull request by Romanov7788**

### CODE REVIEW

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code
You need to separate these words with a space
image

**Review by etojeDenys — CHANGES_REQUESTED** [review]
still not fixed
image

**Comment by Romanov7788** [comment]
I don't understand what I did wrong. There are spaces in the text. I've attached a photo.
Снимок экрана 2024-09-23 в 21 50 16

**Comment by Romanov7788** [comment]
All the tests were passed successfully.
Снимок экрана 2024-09-23 в 21 57 30

**Comment by Romanov7788** [comment]
What do I need to do to change the image in the DEMO LINK?

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+
+inputs.forEach((input) => {
+  const label = document.createElement('label');
+
+  label.classList.add('field-label');
+  label.setAttribute('for', input.id);
+
+  if (input.name === 'firstName' || input.name === 'lastName') {
+    const formattedName =
+      input.name === 'firstName' ? 'First Name' : 'Last Name';
```
Comment (VitaliyBondarenko1982) [inline src/scripts/main.js:13]: it is hardcoded solution. 
try to make more flexible solution.
separation input.name words should work for any words and any amount of words.
Use setAttribute method for changing placeholder.

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Try to fix prev comments.
Label and placeholder should have space between words.
Such case not cover tests, so your tests can be passed.

If will have some questions - feel free to ask in chat.

**Pull request closed without merging**